Makefile

```diff
@@ -35,6 +35,7 @@ lint:
 	  -- index.js
 	$(ESLINT) \
 	  --env node \
+	  --env es6 \
 	  --env mocha \
```

Comment:
The order in which `--env` values are provided isn't significant, but I like `--env es6 --env node --env mocha` as `language-version > engine > framework` seems like the appropriate hierarchy. I love to avoid making arbitrary decisions. :)

Also, please add `--env es3` to the `$(ESLINT) ... -- index.js` command.